PWGCF/TableProducer/filter2Prong.cxx

```diff
@@ -147,7 +148,7 @@ struct Filter2Prong {
     // The main filter outputs the primary MC particles. Here we just resolve the daughter indices that are needed for the efficiency matching.
     for (const auto& r : cfmcparticles) {
       const auto& mcParticle = r.mcParticle_as<HFMCTrack>();
-      if ((mcParticle.flagMcMatchGen() & (1 << aod::hf_cand_2prong::DecayType::D0ToPiK)) == 0 || mcParticle.daughtersIds().size() != 2) {
+      if ((std::abs(mcParticle.flagMcMatchGen()) != o2::hf_decay::hf_cand_2prong::DecayChannelMain::D0ToPiK) || mcParticle.daughtersIds().size() != 2) {
```

Comment:
@jaelpark What was the intention of the sentence Vit replaced? 
I would say they should produce the same result (it is a good replacement) but it would depend on the original intention

Reply:
I learned that `abs` is needed for completeness, so Vit just added that. Otherwise I guess they're equivalent.

Reply:
Thanks!